src/xdocs/config_coding.xml

```diff
@@ -4889,6 +4889,25 @@ if (&quot;something&quot;.equals(x))
         <source>
 &lt;module name=&quot;UnnecessaryParentheses&quot;/&gt;
         </source>
+<p>
+  Which results in the following violations:
+  </p>
+  <source>
```

Comment:
Align `<p>`, text and `<source>` with previous `<p>` and its text. Indentation is lopsided.